starter-bots/scala/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM entelectchallenge/base:2019
+
+RUN apt-get update -y
+
+RUN apt-get install default-jre -y
+RUN apt-get install default-jdk -y
```

Comment:
It should be possible to omit the first line as the JRE is bundled with the JDK.